momento-sdk/src/intTest/java/momento/sdk/ListTest.java

```diff
@@ -914,158 +970,173 @@ public void shouldFailListPushBackWhenNullElement() {
+            target.listPushFront(
+                null, listName, byteArrayValue, null, CollectionTtl.fromCacheTtl()))
         .succeedsWithin(FIVE_SECONDS)
         .asInstanceOf(InstanceOfAssertFactories.type(CacheListPushFrontResponse.Error.class))
         .satisfies(error -> assertThat(error).hasCauseInstanceOf(InvalidArgumentException.class));
 
     // Without ttl specified in method signature
-    assertThat(target.listPushFront(null, listName, byteArrayValue, 100))
+    assertThat(target.listPushFront(null, listName, byteArrayValue, null))
         .succeedsWithin(FIVE_SECONDS)
         .asInstanceOf(InstanceOfAssertFactories.type(CacheListPushFrontResponse.Error.class))
         .satisfies(error -> assertThat(error).hasCauseInstanceOf(InvalidArgumentException.class));
   }
 
+  @SuppressWarnings("DataFlowIssue")
   @Test
   public void shouldFailListPushFrontWhenNullListName() {
     final String stringValue = "val1";
     final byte[] byteArrayValue = "val1".getBytes();
 
     // With ttl specified in method signature
     assertThat(
-            target.listPushFront(cacheName, null, stringValue, 100, CollectionTtl.fromCacheTtl()))
+            target.listPushFront(cacheName, null, stringValue, null, CollectionTtl.fromCacheTtl()))
         .succeedsWithin(FIVE_SECONDS)
         .asInstanceOf(InstanceOfAssertFactories.type(CacheListPushFrontResponse.Error.class))
         .satisfies(error -> assertThat(error).hasCauseInstanceOf(InvalidArgumentException.class));
 
     // Without ttl specified in method signature
-    assertThat(target.listPushFront(cacheName, null, stringValue, 100))
+    assertThat(target.listPushFront(cacheName, null, stringValue, null))
         .succeedsWithin(FIVE_SECONDS)
         .asInstanceOf(InstanceOfAssertFactories.type(CacheListPushFrontResponse.Error.class))
         .satisfies(error -> assertThat(error).hasCauseInstanceOf(InvalidArgumentException.class));
 
     // With ttl specified in method signature
     assertThat(
             target.listPushFront(
-                cacheName, null, byteArrayValue, 100, CollectionTtl.fromCacheTtl()))
+                cacheName, null, byteArrayValue, null, CollectionTtl.fromCacheTtl()))
         .succeedsWithin(FIVE_SECONDS)
         .asInstanceOf(InstanceOfAssertFactories.type(CacheListPushFrontResponse.Error.class))
         .satisfies(error -> assertThat(error).hasCauseInstanceOf(InvalidArgumentException.class));
 
     // Without ttl specified in method signature
-    assertThat(target.listPushFront(cacheName, null, byteArrayValue, 100))
+    assertThat(target.listPushFront(cacheName, null, byteArrayValue, null))
         .succeedsWithin(FIVE_SECONDS)
         .asInstanceOf(InstanceOfAssertFactories.type(CacheListPushFrontResponse.Error.class))
         .satisfies(error -> assertThat(error).hasCauseInstanceOf(InvalidArgumentException.class));
   }
 
+  @SuppressWarnings("DataFlowIssue")
```

Comment:
What is this for?

Reply:
It suppresses the warnings you get when supplying null to a @Nonnull method argument. You also get a warning at the method site but I changed that inspection to a weak warning in Intellij.